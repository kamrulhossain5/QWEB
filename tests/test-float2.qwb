function int main(){
  float a;
  float b;
  float c;
  a = 5.12345678;
  b = -2.12345;
  c = a + b;
  printf(c);
}
