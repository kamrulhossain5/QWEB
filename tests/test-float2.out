3.00001<br>
